### PR TITLE
Small fixes (and rename offer to order...)

### PR DESCRIPTION
**Pull request by da-kami**
Offer was a bad choice of name - I should have known better 😬
Please let me fix it :)

### CODE REVIEW

File: daemon/src/model/cfd.rs [hunk 1]
```diff
@@ -254,9 +254,9 @@ pub struct Cfd {
 }
 
 impl Cfd {
-    pub fn new(cfd_offer: CfdOffer, quantity: Usd, state: CfdState, position: Position) -> Self {
+    pub fn new(cfd_offer: Order, quantity: Usd, state: CfdState, position: Position) -> Self {
```
Comment (thomaseizinger) [inline daemon/src/model/cfd.rs]: This variable still says `cfd_offer`.

**Review by thomaseizinger — APPROVED** [review]

**Review by klochowicz — CHANGES_REQUESTED** [review]
I'm happy with the name change, but there are still many mentions of "offer" - mainly in variable names, inside `Maker.tsx`, `*_cfd_actor.rs` and `routes_*.rs`. Perhaps you could grep for "offer" throughout the codebase?

File: daemon/src/model/cfd.rs (hunk 1 above)
Comment (da-kami) [reply daemon/src/model/cfd.rs]: crap, there are a few that still say it ... - fixing

Comment (da-kami) [reply daemon/src/model/cfd.rs]: done

**Comment by da-kami** [comment]
> I'm happy with the name change, but there are still many mentions of "offer" - mainly in variable names, inside `Maker.tsx`, `*_cfd_actor.rs` and `routes_*.rs`. Perhaps you could grep for "offer" throughout the codebase?

Sry, missed a few (had a filter set that messed that up...) - Should be fixed now. please have another look

**Review by klochowicz — APPROVED** [review]
LGTM! 👍

**Pull request merged**